Library/Formula/rpm.rb

```diff
@@ -52,18 +56,20 @@ def install
         --with-neon=external
         --with-uuid=external
         --with-pcre=external
-        --with-lua=internal
-        --with-syck=internal
+        --with-lua=external
+        --with-syck=external
```

Comment:
I don't really mind if it's default or not as long as they are still vendoring these things in a way that we can use them. I'd be OK with changing all of the above to be `internal` too.